Assets/Code/Sprite/PositionGrid/PositionGrid.cs

```diff
@@ -89,6 +89,31 @@ private bool Move(Character character, GridCoordinates target)
         return validMove;
     }
 
+    public bool MoveUpValid(Character character)
+    {
+        GridCoordinates target = character.Position;
+        target.Y += 1;
+        return BoundCheck(target.X, target.Y);
+    }
+    public bool MoveDownValid(Character character)
+    {
+        GridCoordinates target = character.Position;
+        target.Y -= 1;
+        return BoundCheck(target.X, target.Y);
+    }
+    public bool MoveLeftValid(Character character)
+    {
+        GridCoordinates target = character.Position;
+        target.X -= 1;
+        return BoundCheck(target.X, target.Y);
+    }
+    public bool MoveRightValid(Character character)
+    {
+        GridCoordinates target = character.Position;
+        target.X += 1;
+        return BoundCheck(target.X, target.Y);
+    }
+
```

Comment:
I don't like the idea of duplicating the move validation code from the actual move functions to these new ones. There's probably a better way to structure this entirely. Perhaps have these validity functions inside the functions that will have the  actual movement? They are already returning the BoundCheck() boolean.